jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInvokeRunner.kt

```diff
@@ -168,7 +173,19 @@ class SamInvokeRunner : AsyncProgramRunner<RunnerSettings>() {
                 awsRegion = lambdaSettings.connection.region.id
             ),
             debug = isDebug,
-            runtime = TelemetryRuntime.from(lambdaSettings.runtime.toString()),
+            runtime = TelemetryRuntime.from(
+                when (lambdaSettings) {
+                    is ZipSettings -> {
+                        lambdaSettings.runtime.toString()
+                    }
+                    is ImageSettings -> {
+                        lambdaSettings.imageDebugger.id
+                    }
+                    else -> {
+                        ""
```

Comment:
Should we sue TelemetryRuntime.Unkown?

Reply:
This is a `when` inside of the `from` so passing in the raw value would be equivalent

Reply:
True, itll still become Unknown, and i made it not nullable. Fair enough.